css-images-4/Overview.bs

```diff
@@ -72,7 +72,7 @@ Value Definitions {#values}
 	or <a section href="#gradients">gradient notation</a>.
 	Its syntax is:
 
-	<pre class="prod"><dfn>&lt;image></dfn> = <<url>> | <<image()>> | <<image-set()>> | <<cross-fade()>> | <<element()>> | <<gradient>></pre>
+	<pre class="prod"><dfn>&lt;image></dfn> = <<url>> | <<image()>> | <<image-set()>> | <<cross-fade()>> | <<element()>> | <<gradient>> | <<1d-image>></pre>
```

Comment:
I don't think we have a WG resolution on using 1D images as 2D images, so I wouldn't include that here.

(I also don't think it's a useful idea. If we want stripes() to work in 2D, we should define a 2D stripes function, which can re-orient the stripes as appropriate. It's not worth the implementation/testing/documentation effort to have a stripes() function available in all 2D contexts that can only do horizontal stripes. IMHO)

Reply:
As pointed out in https://github.com/w3c/csswg-drafts/issues/2532#issuecomment-1104836974, the plan is to add 1D images to gradients. So we might end up using that instead of adding it to `<image>` directly.
Let's discuss that in the next call.

Sebastian

Reply:
As discussed, we've split the decision to use 1D images in 2D contexts out of this PR. For reference, see #7241 and #7244.

Sebastian